### PR TITLE
vendor latest c/image

### PR DESCRIPTION
**Pull request by vrothberg**
When copying images and the output is not a tty (e.g., when piping to a
file) print single lines instead of using progress bars.  This avoids
long and hard to parse output.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
CI PR for https://github.com/containers/image/pull/558.
*Don't merge*

**Comment by mtrmac** [comment]
ACK

**Comment by rhatdan** [comment]
LGTM
